pkg/vpcmodel/textOutput.go

```diff
@@ -66,12 +66,14 @@ func (t *TextOutputFormatter) WriteOutput(c1, c2 *VPCConfig,
 		return nil, err
 	}
 	hasStatelessConns := false
+	hasOverApproximatedConn := false
```

Comment:
encapsulation of `hasStatelessConns` and `hasOverApproximatedConn` also relevant here, of course

Reply:
this messaging mechanism is about to be rewritten. am not going to do it in this PR